src/main/java/yevhenii/lostfilmdemo/services/impl/SchedulerServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package yevhenii.lostfilmdemo.services.impl;
+
+import lombok.Getter;
+import org.quartz.*;
+import org.quartz.impl.StdSchedulerFactory;
+import org.springframework.stereotype.Service;
+import yevhenii.lostfilmdemo.quartz.LostfilmRssJob;
+import yevhenii.lostfilmdemo.services.SchedulerService;
+
+import static org.quartz.SimpleScheduleBuilder.simpleSchedule;
+import static org.quartz.TriggerKey.triggerKey;
+
+@Service
+public class SchedulerServiceImpl implements SchedulerService {
+    @Getter
+    static int jobCount;
+    @Getter
+    static int triggerCount;
+
+    public void start() throws SchedulerException {
+        Scheduler scheduler = StdSchedulerFactory.getDefaultScheduler();
```

Comment:
you don't need to do that, you are using starter and it has auto configuration

Reply:
oh..tu, cause i wasting few hours to find how to use di here